docs/MapperConfiguration.md

```diff
@@ -0,0 +1,200 @@
+This `sensorMaxRange` values defines a radiusFilter that removes all points from a scan that is outside of a sphere centered at the robot.
+
+Last but not least, the `sensorMaxRange` also defines an area around the robot where the Voxel Manager operates.
+
+##### Voxel Manager
+The purpose of the voxel manager is to limit the computational complexity of the map maintenance, instead of letting it
+grow with its size.
+A map is divided into voxels and only these cells that are close to the robot are kept in RAM.
+The remaining points are stored on the system's hard drive.
+The local map dimensions are defined as twice the `sensorMaxRange` plus a margin of two cells on each side.
+A single cell size is currently hard-coded to 20 meters.
+As an example, setting the `sensorMaxRange` to 100 meters, the local map stored on the RAM will be a cube with a side of
+$$
+2 \times 100 + 2 \times (2 \times 20)~m = 280~m
+$$
+with the robot located at its center.
+For more details on the long-term map storage, check the [Kilometer-scale autonomous navigation in subarctic forests: challenges and lessons learned](https://norlab.ulaval.ca/publications/field-report-ltr/) paper.
+
+### Mapper Modules
+Mapper Modules are a way to configure what happens when a lidar scan is merged into the map.
+This can include an examination of new points already exist in the map, or removal of dynamic points.
+
+The library implements these filters:
+#### Down-sampling
+
+_PointDistance_
+
+Adds a new descriptor to an existing point cloud or overwrites existing descriptor with the same name.
+
+- **Impact on the number of points:** reduces number of points 
+
+| Parameter       | Description                                                                          | Default value | Allowable range |
+|-----------------|:-------------------------------------------------------------------------------------|:--------------|:----------------|
+| minDistNewPoint | Minimum distance between a new point and its nearest neighbor in the map (in meters) | 0.03          | [0.0, inf)      |
+
+_Octree_
+
+The filter use the efficent spatial representation of the pointcloud by the octree to sub-sample point in each leaf.
+Since the library just provides a wrapper for `libpointmatcher`, check [its docs](https://libpointmatcher.readthedocs.io/en/latest/DataFilters/#octree-grid-filter) for more details.
+
+#### Dynamic points management
+_ComputeDynamics_
+
+Computes a per-point value defining the probability that the points are dynamic, originating from, e.g., walking pedestrians, or falling snow.
+The dynamic filter allows longer operations as dynamic points are not accumulating in the map.
+To identify the dynamic points, ray tracing is used.
+If an incoming scan point is located behind a map point, the probability of this map point being dynamic is increased.
+If this probability surpasses the predefined threshold, it is effectively removed from the map.
```

Comment:
Unless this has been changed, the dynamic point computation does not remove points, it just flags them as dynamic. A post filter must be used to remove points with a probability of being dynamic above a certain threshold.